tests/api.test.js

```diff
@@ -138,7 +154,10 @@ describe("Test /api/", () => {
 
   it("should have proper cache", async () => {
     const { req, res } = faker({}, data);
-    mock.onPost("https://api.github.com/graphql").reply(200, data);
+    mock.onPost("https://api.github.com/graphql")
```

Comment:
This was already in the codebase before your PR, but I don't think this line is needed, right? It should already work without this `mock.onPost` call because of 

https://github.com/anuraghazra/github-readme-stats/blob/cb6550fd163578b735206ff4982b434d6868ca22/tests/api.test.js#L94-L96

and 

https://github.com/anuraghazra/github-readme-stats/blob/cb6550fd163578b735206ff4982b434d6868ca22/tests/api.test.js#L86-L89

But maybe I'm overlooking something. What do you think?

Reply:
if removed the tests start failing. So I guess that those are needed.

Reply:
Actually only the reset is needed. I removed the other mock part.

Reply:
I can remove these lines without the test failing. 🤔 Can you test again?

Reply:
Just lines 157-160. All the other mocks are correct.

Reply:
yep. I pushed a change where those are removed